src/io/flutter/view/InspectorColoredTreeCellRenderer.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.view;
+
+import com.intellij.ui.JBColor;
+import com.intellij.ui.LoadingNode;
+import com.intellij.util.ui.JBUI;
+import com.intellij.util.ui.UIUtil;
+import com.intellij.util.ui.tree.WideSelectionTreeUI;
+import org.jetbrains.annotations.NotNull;
+
+import javax.accessibility.AccessibleContext;
+import javax.swing.*;
+import javax.swing.tree.TreeCellRenderer;
+import java.awt.*;
+
+abstract class InspectorColoredTreeCellRenderer extends MultiIconSimpleColoredComponent implements TreeCellRenderer {
```

Comment:
Boilerplate to make a TreeCellRenderer out of a MultiIconSimpleColoredComponent
based on the existing class for SimpleColoredComponent but I have lost the diff tracking what I really changed for this small class.  Generally assume this is all boilerplate although perhaps I hacked on the code to set the background a bit.